proto/api.proto

```diff
@@ -974,6 +979,17 @@ message GetQuotesStreamRequest {
   repeated TokenPair tokenPairs = 2;
 }
 
+message GetSwapsStreamRequest {
+  repeated string markets = 1;
```

Comment:
should probably add `repeated Project projects` here as well –– for now only Raydium should be supported, but would be good to get ahead of defining this endpoint

Reply:
let's add, thought about that